rpc/trace_test.go

```diff
@@ -157,11 +163,15 @@ func TestTraceTransaction(t *testing.T) {
 		require.NoError(t, json.Unmarshal(json.RawMessage(vmTraceJSON), vmTrace))
 		consumedGas := []*felt.Felt{new(felt.Felt).SetUint64(1)}
 		overallFee := []*felt.Felt{new(felt.Felt).SetUint64(1)}
+		numSteps := uint64(0)
 		mockVM.EXPECT().Execute([]core.Transaction{tx}, []core.Class{declaredClass.Class}, []*felt.Felt{},
-			&vm.BlockInfo{Header: header}, gomock.Any(), &utils.Mainnet, false, false, false, true).Return(overallFee, consumedGas, []vm.TransactionTrace{*vmTrace}, nil)
+			&vm.BlockInfo{Header: header}, gomock.Any(), &utils.Mainnet, false, false,
+			false, true).Return(overallFee, consumedGas, []vm.TransactionTrace{*vmTrace}, numSteps, nil)
 
-		trace, err := handler.TraceTransaction(context.Background(), *hash)
+		trace, httpHeader, err := handler.TraceTransaction(context.Background(), *hash)
 		require.Nil(t, err)
+		require.NotNil(t, httpHeader)
+		require.NotEmpty(t, httpHeader.Get(rpc.ExecutionStepsHeader))
```

Comment:
nit: we can check it's 0 right?

Reply:
yes, though the idea was, it wasn't being tested for or expected.
So as log as it returned a value that was enough for a couple of the tests.

updated now though.

Also most of the tests with t.Skip where apparently previously done purposefully.

Reply:
this has been fixed in main and merged into this branch